### ISSUE TITLE
ES6 syntax not working in package

### ISSUE THREAD

**Issue by oznekenzo** [3box/profile-hover#27]
Not sure what happened in the setup / if es6+ was ever enabled

this is needed / helpful for creating functionality that prevents hover from going outside window

**Comment by raksooo** [comment]
Hi!

Which es6+ features is it that dosn’t work? The React component and the widget both uses a bit of es6+ like arrow functions, destructuring, async/await. Maybe polyfill is needed?

**Comment by oznekenzo** [comment]
I wasn't able to use arrow functions in the profile component, but thought it was really weird because I saw an async function right above.

**Comment by raksooo** [comment]
Weird! in ProfileHover.jsx? If I remember correctly there’s no arrow functions in that component but the components it imports from html.jsx uses arrow functions. The react component uses class syntax though.  I’ll try it out when I find time for it :)

**Comment by oznekenzo** [comment]
Yes very weird. I found a workaround so not critical at the moment
